infra/openshift-manifests/reply-service/service.yaml

```diff
@@ -31,6 +31,38 @@ spec:
         - containerPort: 8080
 ---
 apiVersion: v1
+kind: Pod
+metadata:
+  name: reply-service-3
+  namespace: ai-demo
+  labels:
+    app: reply-service
+    instance: reply-service-3
+spec:
+  containers:
+    - name: reply-service
+      image: quay.io/kevent-mesh/ai-demo-reply-service:main
+      imagePullPolicy: Always
+      ports:
+        - containerPort: 8080
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: reply-service-4
+  namespace: ai-demo
+  labels:
+    app: reply-service
+    instance: reply-service-4
+spec:
+  containers:
+    - name: reply-service
+      image: quay.io/kevent-mesh/ai-demo-reply-service:main
+      imagePullPolicy: Always
+      ports:
+        - containerPort: 8080
+---
+apiVersion: v1
```

Comment:
Manually scaled up the reply service to 4 pods.
Let's keep it this way IMO, until we find what the issue is. If the bottleneck is Minio, having more reply services won't help.